codespan-reporting/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.2] - 2020-10-XX
+
 There is now a [code of conduct](https://github.com/brendanzab/codespan/blob/master/CODE_OF_CONDUCT.md)
 and a [contributing guide](https://github.com/brendanzab/codespan/blob/master/CONTRIBUTING.md).
```

Comment:
Might be good to note that we have skipped `v0.10.0` and `v0.10.1` to get us back in sync with the `codespan-lsp` version